docs/2-how-it-works/pricing.md

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 2
+---
+
+# Pricing
+Serverless is about simplicity in every dimension, including pricing! Momento costs $0.15/GB transferred. That's it!
+
+Your first 50 GB are free each month—and you don't even need a credit card to get started.
```

Comment:
I still wish we could mention the free part before the non-free part.  or bold this or something.  but not a blocker for getting this out the door.

Reply:
I agree I think. Let me get this one merged and can start more focused discussion on that in slack and address in follow up.